docs/core/concepts/schedules.md

```diff
@@ -2,9 +2,32 @@
 
 ## Overview
 
-Prefect assumes that flows can be run at any time, for any reason. However, it is often useful to automate flow runs at certain times. For this purpose, Prefect provides a versatile `Schedule` object.  Note that schedules can be attached to Flows via the `schedule` keyword argument, as in [this example](https://docs.prefect.io/core/examples/daily_github_stats_to_airtable.html).
+Prefect assumes that flows can be run at any time, for any reason. However, it is often useful to automate flow runs at certain times. Prefect assumes that flows can be run at any time, for any reason. However, it is often useful to automate flow runs at certain times. Simple schedules can be attached to Flows via the `schedule` keyword argument. For more detailed or complex schedules, Prefect provides a versatile `Schedule` object.
```

Comment:
Looks like the first two sentences got copy / pasted twice

Reply:
Argh! Updated.